src/lib/y2storage/storage_feature.rb

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) [2016-2017,2019-2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "storage"
+
+module Y2Storage
+  #
+  # Class representing the storage features defined by libstorage-ng
+  #
+  # In libstorage-ng each feature is represented by a bit-mask identified
+  # by a constant with name UF_XXX (with XXX being the name of the feature).
+  # A set of features is represented by a bit-field of those masks.
+  #
+  # Thus, one of those bit-fields is used to specify which features are
+  # used in the current target machine's storage setup. Additionally, a similar
+  # bit-field is used to report features that have been detected in the system
+  # but that cannot be probed because some command in missing.
```

Comment:
"is missing"

Reply:
Fixed in a later commit already added to this PR.